dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -115,6 +115,15 @@ def test_diff_primitive_compilers(self):
 
         compiler1 = col_pro_compilers.ColumnPrimitiveTypeProfileCompiler(data1)
         compiler2 = col_pro_compilers.ColumnPrimitiveTypeProfileCompiler(data2)
+
+        def get_welch_df(var1, n1, var2, n2):
+            return (var1 / n1 + var2 / n2) ** 2 \
+                   / ((var1 / n1) ** 2 / (n1 - 1)
+                      + (var2 / n2) ** 2 / (n2 - 1))
```

Comment:
I noticed you have this function multiple times, should probably move this out to a util so you can call it a bunch without having to define it

Reply:
since tests don't change, we should expect the results to be unchanged and instead of using a function should just be the values.

Reply:
or are the values in these cases two extreme to reduce?